data_steward/cdr_cleaner/cleaning_rules/deid/concept_suppression.py

```diff
@@ -146,14 +146,16 @@ def get_query_specs(self, *args, **keyword_args) -> query_spec_list:
         sandbox_queries = [
             self.get_sandbox_query(table_name)
             for table_name in self.affected_tables
-            if get_concept_id_fields(table_name)
+            if (get_concept_id_fields(table_name) and
+                has_domain_table_id(table_name))
         ]
```

Comment:
Could we do this once in initialization of `AbstractConceptSuppression`?  Like:

```
# in class initialization
affected_tables = [table_name for table_name in affected_tables 
                               if (get_concept_id_field(table_name) and has_domain_table_id(table_name))]

super().__init__(issue_numbers=issue_numbers,
                         description=description,
                         affected_datasets=affected_datasets,
                         project_id=project_id,
                         dataset_id=dataset_id,
                         sandbox_dataset_id=sandbox_dataset_id,
                         affected_tables=affected_tables)


# then these lines would just be
for table_name in self.affected_tables
```

Primary benefits:  one place  to update instead of two and `self.affected_tables` reflects them tables that are actually affected.